### ISSUE TITLE
Postgres ?

### ISSUE THREAD

**Issue by hopewise** [rails-lambda/lamby#161]
Reading document here https://lamby.cloud/docs/database, seems there is no example about using Rails with Postgres

I tried to use Dockerfile as:

```
FROM ruby:3.2.0-buster

# Install dependencies needed for PostgreSQL
RUN apt-get update && \
    apt-get install -y postgresql-client libpq-dev

```

but when running the app in lambda, I got error:

```
could not connect to server: No such file or directory Is the server running locally and accepting connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```

Is there any example of Rails using Postgres with Lamby based project?

**Comment by metaskills** [comment]
No, but technically this is not in the Lamby camp since this is "just Docker" which is a high level way of saying it is just 1) Rails, Ubuntu, Docker, Docker Compose, & Devcontainers. Meaning all the info out there on running Postgress with these technologies is out there. 

That said, you are on the right path, but remember you will need this in both the .devcontainer docker file and the root one shipped to prod. You will also need to update your database.yml and swap out the mysql service with a pg one in your devcontainer compose file.
